parts/kubernetesagentcustomdata.yml

```diff
@@ -135,7 +135,8 @@ write_files:
             --allow-privileged=true \
             --enable-server \
             --enable-debugging-handlers \
-            --config=/etc/kubernetes/manifests \
+            --feature-gates=Accelerators=true \
```

Comment:
If the machine doesn't have GPU, does this flag have any effect?  We want to avoid affecting existing customers who do not want GPU support.